README.md

```diff
@@ -22,9 +22,11 @@ headers, and XML registry.
 
 ### Assembler, binary parser, and disassembler
 
-* Based on SPIR-V 1.0 Revision 3.
-  * Supports GLSL std450 extended instructions.
-  * Supports OpenCL extended instructions.
+* Based on SPIR-V 1.1 Rev 1
+* Support for extended instruction sets:
+  * GLSL std450 (v1.0)
```

Comment:
The extended instruction instruction sets have versions to.  And why write the version differently from SPIRV?
GLSL std450 is at 1.0 Rev 3
OpenCL is at 1.0 Rev 2    (even though the header file doesn't advertise it.)
https://www.khronos.org/registry/spir-v/specs/1.0/OpenCL.ExtendedInstructionSet.100.html